frontend/src/pages/NewRun.tsx

```diff
@@ -886,21 +892,21 @@ export class NewRun extends Page<{ namespace?: string }, NewRunState> {
       return true;
     } catch (err) {
       const errorMessage = await errorToMessage(err);
-      this.showErrorDialog('Failed to upload pipeline', errorMessage);
+      this.showErrorDialog(t('common:pipelineUploadError'), errorMessage);
       return false;
     }
   }
 
   private async _prepareFormFromEmbeddedPipeline(embeddedRunId: string): Promise<void> {
     let embeddedPipelineSpec: string | null;
     let runWithEmbeddedPipeline: ApiRunDetail;
-
+    const { t } = this.props;
     try {
       runWithEmbeddedPipeline = await Apis.runServiceApi.getRun(embeddedRunId);
       embeddedPipelineSpec = RunUtils.getWorkflowManifest(runWithEmbeddedPipeline.run);
     } catch (err) {
       await this.showPageError(
-        `Error: failed to retrieve the specified run: ${embeddedRunId}.`,
+        `${t('errorRetrieveSpecRun')}: ${embeddedRunId}.`,
```

Comment:
Same comment as before, error message could probably have a way to take argument, and therefore include the embeddedRunId in it's message directly.

Reply:
{      
    "key1": "{${embeddedRunId}}",
     "key2": "Error: failed to retrieve the specified run:{{val}}"
}
i18next.t('key2', {val: '$t(key1)'});
Need to find a way to pass the value of embeddedRunId, didnt worked in this way. And if we decide to use the nested or interpolation function, must change here and many mores places.